src/Sylius/Bundle/CoreBundle/Form/Type/CatalogPromotionScope/ForTaxonsScopeConfigurationType.php

```diff
@@ -31,7 +31,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             'label' => 'sylius.ui.taxons',
             'multiple' => true,
             'required' => false,
-            'choice_name' => 'name',
+            'choice_name' => 'fullname',
```

Comment:
This can be removed, because that's the default value in `Sylius\Bundle\TaxonomyBundle\Form\Type\TaxonAutocompleteChoiceType`